code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -103,6 +103,12 @@
 	for(var/obj/item/F in H)
 		if(F.light_range > 0)
 			extinguishItem(F)
+	if(ishuman(H))
```

Comment:
the var letters on this are badddd (not your fault).

I'd suggest adjust the original argument name to `var/mob/living/L`, and going from there; reason being, consistnetly `L` is used for living, and `H` for human, and `M` for generic mob (or as a lump of all mobs together).